config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get      '/login',   to:  'sessions#new'
+  post     '/login',   to:  'sessions#create'
+  delete   '/logout',  to:  'sessions#destroy'
+  resources :users, only: [:new, :create, :show] 
+  resources :events
```

Comment:
### OPTIONAL CHANGE
- [ ] Refactor this line to the routes only for used actions👇 
```suggestion
  resources :events, only: [:new, :create, :show, :index]`
```